src/extension/iSamples_rangeFacet.js

```diff
@@ -0,0 +1,146 @@
+import PropTypes from 'prop-types';
+import React from "react";
+import cx from "classnames";
+
+import RangeSlider from "./iSamples_rangeSlider";
```

Comment:
We didn't actually change anything in the slider itself, but it isn't exported from `solr-faceted-search-react`, so I had to copy it in here.  I suppose I could investigate just exporting out of there, too, but I'm not sure if it's possible/easy.

Reply:
Yeah, I did the same thing for the icon/search.js.